packages/components/rollup-styles.config.js

```diff
@@ -18,12 +18,16 @@ const generateStylesEntryFile = () => {
   const stylesExports = stylesPaths
     .map((utilPath) => {
       const isCommonComponent = utilPath.includes('/components/common/');
-      const styleExport = isCommonComponent
-        ? `*`
-        : `{ getComponentCss as get${pascalCase(/([-a-z]+)-styles\.ts/.exec(utilPath)[1])}Css }`;
-      const fromPath = `./components${utilPath.replace(componentsDir, '').replace(/\.ts$/, '')}`;
+      const isHeadlineComponent = utilPath.includes('/components/headline/');
```

Comment:
Seems to be the wrong place to have component specific tweaks in a build config.